packages/fx-core/src/plugins/resource/localdebug/tasks.ts

```diff
@@ -72,8 +72,15 @@ export function generateSpfxTasks(): Record<string, unknown>[] {
   return [
     {
       label: "dependency check",
-      type: "shell",
-      command: "echo ${command:fx-extension.validate-spfx-dependencies}",
+      type: "process",
+      command: "${command:fx-extension.validate-spfx-dependencies}",
+      presentation: {
```

Comment:
Try "exit ${command:...}" to remove the `presentation` section.

Reply:
Great point. Using `exit ${command:xxx}` will be more smooth.
Updated. Please note that we have to return string "1" rather than number 1, otherwise there will be an error:
![B@GJX 0_EQBVFCR _WKA$5V](https://user-images.githubusercontent.com/37978464/145537304-1e3c68e4-c72c-4199-9545-9477ecf8d948.png)